### PR TITLE
Nested Ternary Operator

### PR DESCRIPTION
**Pull request by Anjali-1801**
An example of a nested ternary operator along with the explanation.

### CODE REVIEW

File: Operators/nested_ternary_operator.c [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+int main()
+{
+    int var= 75;
```
Comment (sarvesh-0) [inline Operators/nested_ternary_operator.c:4]: int var =75, var2 =56, num; // replace  line 4-6 by this.

That's a usecase for comma operator.

File: Operators/nested_ternary_operator.c [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    int var2=56;
+    int num;
+    num= sizeof(var) ? (var2>23 ? ((var==75) ? 'A' : 0) :0) :0;
+    printf("%d", num);
+    return 0;
+}
+
+/* 65 is the answer.
+
+Explanation:
```
Comment (sarvesh-0) [inline Operators/nested_ternary_operator.c:14]: Nice explaination!

File: Operators/nested_ternary_operator.c (hunk 1 above)
Comment (Anjali-1801) [reply Operators/nested_ternary_operator.c]: Made the changes sir, should I merge the pull request now?

File: Operators/nested_ternary_operator.c (hunk 2 above)
Comment (Anjali-1801) [reply Operators/nested_ternary_operator.c:12]: Thankyou sir :)

File: Operators/nested_ternary_operator.c (hunk 1 above)
Comment (sarvesh-0) [reply Operators/nested_ternary_operator.c]: I assigned Ritika as a reviewer - let her review and merge. 
General practice: once should avoid merging his/her own pull request.
You can from your end as well, assign one of the other contributors as a reviewer from next time. Doing this just to put in place the process. Feel free to merge any other PR that you feel is okay to merge.

Comment (Anjali-1801) [reply Operators/nested_ternary_operator.c]: Okay Sir.

Comment (Ritika014) [reply Operators/nested_ternary_operator.c]: Sir, I have seen the changes and will merge the pull request now.
Also, I really liked the explanation that Anjali has provided.

**Pull request merged**